plugins/inventory/cobbler.py

```diff
@@ -56,6 +56,15 @@
         default: []
         elements: str
         version_added: 4.4.0
+      inventory_hostname:
+        description:
+          - What to use for the ansible inventory hostname.
+          - By default the networking hostname is used if defined, otherwise the dns name of the management or first non-static interface.
+          - If set to I(system), the cobbler system name is used.
+        type: str
+        choices: [ 'hostname', 'system' ]
+        default: hostname
+        version_added: 6.7.0
```

Comment:
```suggestion
        version_added: 7.0.0
```

Reply:
Looks like 7.0.0 was released, so I changed to 7.1.0